dependencies.gradle

```diff
@@ -101,3 +101,6 @@ libraries["rhino"] = "org.mozilla:rhino:1.7.14"
 libraries["slf4j"] = "org.slf4j:slf4j-api:1.7.36"
 libraries["spotbugs"] = "com.github.spotbugs:spotbugs-annotations:4.7.1"
 libraries["swagger"] = "io.swagger:swagger-annotations:1.6.6"
+libraries["resteasyValidator"] = "org.jboss.resteasy:resteasy-validator-provider:4.4.3.Final"
+libraries["javaxElApi"] = "javax.el:javax.el-api:3.0.0"
```

Comment:
We don't actually need to add the javax.el-api library at all. It looks like hibernate-validator is using the library bundled within Tomcat itself, so let's remove it from both gradle and pom.xml

Reply:
done